ipatests/test_integration/test_replica_promotion.py

```diff
@@ -437,6 +437,56 @@ def test_renewal_master_with_csreplica_manage(self):
         self.assertCARenewalMaster(master, replica.hostname)
         self.assertCARenewalMaster(replica, replica.hostname)
 
+    def test_replica_concheck(self):
+        """Test cases for ipa-replica-conncheck command
+
+        Following test cases would be checked:
+        - when called with --principal (it should then prompt for a password)
+        - when called with --principal / --password
+        - when called without principal and password but with a kerberos TGT,
+          kinit admin done before calling ipa-replica-conncheck
+        - when called without principal and password, and without any kerberos
+          TGT (it should default to principal=admin and prompt for a password)
+        """
```

Comment:
Is there any related issue for this ?

Reply:
No.

Reply:
Please create upstream issue and link it into commit message.

Reply:
done